src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
```suggestion
    private final Random random = new Random();
    
    public static String getRandomColor() {
        int index = random.nextInt(Color.values().length);
```
It is better to make the Random object a class field so that it does not have to be created every time you call a method